docs/404.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: We're sorry, we can't find the page you requested
-description: The URL might be misspelled or the page you're looking for might no longer be available.
+description: 'The URL might be misspelled or the page you're looking for might no longer be available.'
```

Comment:
FWIW, the single quotes aren't always necessary, but it's a good practice to use them.